diff --git a/library/Zend/Form/Element/File.php b/library/Zend/Form/Element/File.php
index 0b81b18ac..11764a3f7 100644
--- a/library/Zend/Form/Element/File.php
+++ b/library/Zend/Form/Element/File.php
@@ -643,7 +643,7 @@ class Zend_Form_Element_File extends Zend_Form_Element_Xhtml
     {
         if (!is_numeric($setting)) {
             $type = strtoupper(substr($setting, -1));
-            $setting = (integer) substr($setting, 0, -1);
+            $setting = (int) substr($setting, 0, -1);

             switch ($type) {
                 case 'K' :
diff --git a/library/Zend/Memory/Manager.php b/library/Zend/Memory/Manager.php
index aa10ab43e..0947ab1d2 100644
--- a/library/Zend/Memory/Manager.php
+++ b/library/Zend/Memory/Manager.php
@@ -169,7 +169,7 @@ class Zend_Memory_Manager

         $memoryLimitStr = trim(ini_get('memory_limit'));
         if ($memoryLimitStr != ''  &&  $memoryLimitStr != -1) {
-            $this->_memoryLimit = (integer)$memoryLimitStr;
+            $this->_memoryLimit = (int)$memoryLimitStr;
             switch (strtolower($memoryLimitStr[strlen($memoryLimitStr)-1])) {
                 case 'g':
                     $this->_memoryLimit *= 1024;
diff --git a/library/Zend/Paginator.php b/library/Zend/Paginator.php
index b0b9df7ff..563dd6a11 100644
--- a/library/Zend/Paginator.php
+++ b/library/Zend/Paginator.php
@@ -1077,7 +1077,7 @@ class Zend_Paginator implements Countable, IteratorAggregate
      */
     protected function _calculatePageCount()
     {
-        return (integer) ceil($this->getTotalItemCount() / $this->getItemCountPerPage());
+        return (int) ceil($this->getTotalItemCount() / $this->getItemCountPerPage());
     }

     /**
