--- a/vendor/shardj/zf1-future/library/Zend/Form/Element.php
+++ b/vendor/shardj/zf1-future/library/Zend/Form/Element.php
@@ -595,7 +595,7 @@ class Zend_Form_Element implements Zend_Validate_Interface
      * @param  string $key
      * @return void
      */
-    protected function _filterValue(&$value, $key)
+    protected function _filterValue(&$value, &$key)
     {
         foreach ($this->getFilters() as $filter) {
             $value = $filter->filter($value);
