--- a/library/Zend/Form/Element.php
+++ b/library/Zend/Form/Element.php
@@ -601,7 +601,7 @@ class Zend_Form_Element implements Zend_Validate_Interface
      * @param  string $key
      * @return void
      */
-    protected function _filterValue(&$value, $key)
+    protected function _filterValue(&$value, &$key)
     {
         foreach ($this->getFilters() as $filter) {
             $value = $filter->filter($value);
@@ -618,7 +618,9 @@ class Zend_Form_Element implements Zend_Validate_Interface
         $valueFiltered = $this->_value;
 
         if ($this->isArray() && is_array($valueFiltered)) {
-            array_walk_recursive($valueFiltered, [$this, '_filterValue']);
+            array_walk_recursive($valueFiltered, function (&$val, $key) {
+                $this->_filterValue($val, $key);
+            });
         } else {
             $this->_filterValue($valueFiltered, $valueFiltered);
         }
